helsingborg-stad/app-mitt-helsingborg
# Feat/case screen

## Explain the changes you’ve made

_Warning: Massive PR incoming_
- Updated the old case overview with new design and some logic.
- Changed the order of bottom navigation to follow the figma design.
- Changed home screen to Case Overview instead of the current dev list of forms.
- Various design fixes.
- Handles case status changes: ongoing, submitted and if EKB cases is empty.
- Created a Case Summary view that can be entered when a case is "submitted". 
- Displays list of Administrators(handläggare) in case summary if the data is provided in case data.

The case overview + summary till needs some more logic and design fixes, but I think that can be implemented later. 

## Explain why these changes are made

Because of high priority

## Explain your solution

Lots of the case views navigation and logic was already implemented and I have just tweaked it a lot. 

## How to test the changes?

1. Delete all cases from your AWS account.
2. Login to the app and watch the new "Home screen": case overview.
3. Then you should see a card with a button that starts a new EKB case.
4. Start the form and then exit.
5. Now the card should be updated with a new ongoing status.
6. Click to continue the form and then finish it.
7. Now the card should be updated with the submitted status.
8. Click on the button to show the application.
9. Now you should be redirected to the case summary view.

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.

## Anyhting else?
Found bug: If you continue an already started case, the current step value go bananas and often redirects to another part of the form. Maybe it's an easy fix but I haven't got time to look at it.
